src/Client/Settings.java

```diff
@@ -404,6 +406,15 @@ public static void definePresets(Properties props) {
     ATTACK_ALWAYS_LEFT_CLICK.put("all", true);
     ATTACK_ALWAYS_LEFT_CLICK.put(
         "custom", getPropBoolean(props, "bypass_attack", ATTACK_ALWAYS_LEFT_CLICK.get("default")));
+    
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("vanilla", false);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("vanilla_resizable", false);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("lite", false);
+    KEEP_MAGIC_PRAYER_TAB_CURSOR.put("default", false);
```

Comment:
Probably should be default true. Otherwise, in PVP, players more knowledgeable about our client options will have the advantage, which is not a good mechanic.

I would even possibly have "lite" be true.

Reply:
fixed